### PR TITLE
Minor fix to avoid faulting on mode= param

### PR DESCRIPTION
**Pull request by pradh**
i.e., fix https://autopush.datacommons.org/explore/#q=Population%20by%20age%20in%20the%20united%20states&mode=strict

### CODE REVIEW

File: static/js/utils/app/explore_utils.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ export function getTopics(
   pageMetadata: SubjectPageMetadata,
   placeUrlVal: string
 ): Item[] {
+  if (_.isEmpty(pageMetadata?.childTopics)) {
```
Comment (chejennifer) [inline static/js/utils/app/explore_utils.ts:43]: could there be a case where there are no childTopics but there are peerTopics or parentTopics? Which, in that case, should we just do `topics = [].concat(pageMetadata?.childTopics || []).concat(... || [])...`

**Review by chejennifer — COMMENTED** [review]
nice, one question though

File: static/js/utils/app/explore_utils.ts (hunk 1 above)
Comment (pradh) [reply static/js/utils/app/explore_utils.ts:43]: good catch.  ok, can I do a good'ol for loop though?  This whole `.concat(xyz || [])` feels so ugly, or is it the JS way? :)

Comment (chejennifer) [reply static/js/utils/app/explore_utils.ts:43]: haha yes, you can do a for loop

Comment (pradh) [reply static/js/utils/app/explore_utils.ts:43]: I went with your js expert suggestion after all :), the need to distinguish between null vs. empty makes it more verbose I realize now.

**Review by chejennifer — APPROVED** [review]
thanks for the fix!

**Comment by pradh** [comment]
Thanks for the review!

**Pull request merged**